crates/core/migrations/2022-01-10-200616_create_requested_creators/up.sql

```diff
@@ -0,0 +1,5 @@
+create table creators (
+  address bytea primary key not null,
+  created_at              timestamp not null default now(),
+  updated_at              timestamp null
```

Comment:
```suggestion
  address     bytea     primary key not null,
  created_at  timestamp not null default now(),
  updated_at  timestamp null
```

Reply:
Actually, additional note on this - do we need the migration if we're saving this table for later?